modules/samples/artifacts/FileEventListener/FileEventListenerSample.siddhi

```diff
@@ -0,0 +1,55 @@
+@App:name('FileEventListenerSample')
+
+@App:description('Identify whether file or folder has been created, modified or deleted in the given uri and throw events.')
+
+/*
+Purpose:
+    This application demonstrates how to configure WSO2 Streaming Integrator Tooling to receive events via
+    fileeventlistener on creation, modification or deletion of a file or directory.
+
+Prerequisites:
+    1) Save this sample.
+    2) Create a folder named 'sweet-factory-logs'.
+    3) Add a file named 'test.txt' in the 'sweet-factory-logs' .
+
+Executing the Sample:
+    1) Start the Siddhi application by clicking on 'Run'
+
+    2) If the Siddhi application starts successfully, the following message would be shown on the console
+        * FileEventListenerSample.siddhi - Started Successfully!
+
+    Notes:
+    If you edit this application while it's running, stop the application -> Save -> Start.
+
+Testing the Sample:
+    1) Perform the following actions in 'sweet-factory-logs' folder that you have already created. These actions will
+    create events.
+            I. Create a new file named as test.txt
+            II. Add new lines in the test.txt
+            III. Delete the file test.txt
+
+
+Viewing the Results:
+    You will see an event similar to the following when you start the Siddhi app.
+    INFO {io.siddhi.core.stream.output.sink.LogSink} - FileEventListenerSample : ResultStream : Event{timestamp=1584442060224,
+    data=[/path/to/sweet-factory-logs/test.txt, test.txt, created], isExpired=false}
+
+    For each action performed in the folder (I, II, III) a new event will be created. The outputs on the console will be similar to the following.
+
+    INFO {io.siddhi.core.stream.output.sink.LogSink} - FileEventListenerSample : ResultStream : Event{timestamp=1584442060224,
+    data=[/path/to/sweet-factory-logs/test.txt, test.txt, modifyingCompleted], isExpired=false}
+    INFO {io.siddhi.core.stream.output.sink.LogSink} - FileEventListenerSample : ResultStream : Event{timestamp=1584442060224,
+    data=[/path/to/sweet-factory-logs/test.txt, test.txt, removed], isExpired=false}
+    Note how the file path, file name and status of a file or folder changes with every action you make in the files
+    given in file.name.list
+*/
+
+@source(type = 'fileeventlistener', uri = "/path/to/sweet-factory-logs/", file.name.list = "test.txt")
```

Comment:
in file-io, it's dir.uri. not uri.
https://github.com/siddhi-io/siddhi-io-file/blob/master/component/src/main/java/io/siddhi/extension/io/file/FileHandler.java#L67
could you please check the sample is working? And make the necessary changes?

Reply:
fixed in https://github.com/wso2/streaming-integrator-tooling/pull/72/commits/e5e7ced0d1f93f1d31ca9f8416106b0c20bccae2